### PR TITLE
Adding CDN properties for Organization Branding images

### PR DESCRIPTION
**Pull request by nkimaina**

### CODE REVIEW

File: api-reference/beta/api/organizationalbrandingproperties-update.md [hunk 1]
```diff
@@ -51,6 +51,7 @@ In the request body, supply the values for relevant fields that should be update
 |bannerLogo|Stream|A banner version of your company logo which appears appears on the sign-in page. .png or .jpg no larger than 36x245px. We recommend using a transparent image with no padding around the logo.|
 |signInPageText|String|Text that appears at the bottom of the sign-in box. You can use this to communicate additional information, such as the phone number to your help desk or a legal statement. This text must be Unicode and not exceed 1024 characters.|
 |squareLogo|Stream|Square version of your company logo. This appears in Windows 10 out-of-box (OOBE) experiences and when Windows Autopilot is enabled for deployment. .png or .jpg no larger than 240x240px and no more than 10kb in size. We recommend using a transparent image with no padding around the logo.|
+|squareLogoDark|Stream|Dark square version of your company logo. This appears in Windows 10 out-of-box (OOBE) experiences and when Windows Autopilot is enabled for deployment. .png or .jpg no larger than 240x240px and no more than 10kb in size. We recommend using a transparent image with no padding around the logo.|
```
Comment (FaithOmbongi) [inline api-reference/beta/api/organizationalbrandingproperties-update.md:54]:
```suggestion
|bannerLogo|Stream|A banner version of your company logo which appears on the sign-in page. Allowed types are PNG or JPEG no larger than 36 × 245 pixels. We recommend a transparent image with no padding around the logo.|
|signInPageText|String|Text that appears at the bottom of the sign-in box. You can use this to communicate additional information, such as the phone number to your help desk or a legal statement. This text must be Unicode and not exceed 1024 characters.|
|squareLogo|Stream|Square version of your company logo that appears in Windows 10 out-of-box experiences (OOBE) and when Windows Autopilot is enabled for deployment. Allowed types are PNG or JPEG no larger than 240 ⅹ 240 pixels and no more than 10 KB in size. We recommend a transparent image with no padding around the logo.|
|squareLogoDark|Stream| A dark square version of your company logo that appears in Windows 10 out-of-box experiences (OOBE) and when Windows Autopilot is enabled for deployment. Allowed types are PNG or JPEG no larger than 240 ⅹ 240 pixels and no more than 10 KB in size. We recommend a transparent image with no padding around the logo. |
```

File: api-reference/beta/api/organizationalbrandingpropertieslocalization-create.md [hunk 2]
```diff
@@ -55,6 +55,7 @@ In the request body, supply the values for relevant fields that should be update
 |bannerLogo|Stream|A banner version of your company logo which appears appears on the sign-in page. .png or .jpg no larger than 36x245px. We recommend using a transparent image with no padding around the logo.|
 |signInPageText|String|Text that appears at the bottom of the sign-in box. You can use this to communicate additional information, such as the phone number to your help desk or a legal statement. This text must be Unicode and not exceed 1024 characters.|
 |squareLogo|Stream|Square version of your company logo. This appears in Windows 10 out-of-box (OOBE) experiences and when Windows Autopilot is enabled for deployment. .png or .jpg no larger than 240x240px and no more than 10kb in size. We recommend using a transparent image with no padding around the logo.|
+|squareLogoDark|Stream| A dark square version of your company logo. This appears in Windows 10 out-of-box (OOBE) experiences and when Windows Autopilot is enabled for deployment. .png or .jpg no larger than 240x240px and no more than 10kb in size. We recommend using a transparent image with no padding around the logo. |
```
Comment (FaithOmbongi) [inline api-reference/beta/api/organizationalbrandingpropertieslocalization-create.md:58]:
```suggestion
|bannerLogo|Stream|A banner version of your company logo which appears on the sign-in page. Allowed types are PNG or JPEG no larger than 36 × 245 pixels. We recommend a transparent image with no padding around the logo.|
|signInPageText|String|Text that appears at the bottom of the sign-in box. You can use this to communicate additional information, such as the phone number to your help desk or a legal statement. This text must be Unicode and not exceed 1024 characters.|
|squareLogo|Stream|Square version of your company logo that appears in Windows 10 out-of-box experiences (OOBE) and when Windows Autopilot is enabled for deployment. Allowed types are PNG or JPEG no larger than 240 ⅹ 240 pixels and no more than 10 KB in size. We recommend a transparent image with no padding around the logo.|
|squareLogoDark|Stream| A dark square version of your company logo that appears in Windows 10 out-of-box experiences (OOBE) and when Windows Autopilot is enabled for deployment. Allowed types are PNG or JPEG no larger than 240 ⅹ 240 pixels and no more than 10 KB in size. We recommend a transparent image with no padding around the logo. |
```

File: api-reference/beta/resources/organizationalbrandingproperties.md [hunk 3]
```diff
@@ -68,6 +74,7 @@ The following is a JSON representation of the resource.
   "id": "String (identifier)",
   "signInPageText": "String",
   "squareLogo": "Stream",
+  "squareLogoDark": "Stream",
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/organizationalbrandingproperties.md:77]: Add all new properties here.

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
In the CREATE API:

1. **Request body** section, add the sentence:

"_In the request body, supply a JSON representation of organizationalBrandingProperties object. The following table lists the properties that are required when you create the default branding._"

2. Are these additional properties required? @AlexanderMars, looping you in to confirm the existing properties.
3. On this note, for required properties, add "Required" to their descriptions in organizationalbrandingproperties.md.

**Review by davidmu1 — APPROVED** [review]

File: api-reference/beta/api/organizationalbrandingproperties-create.md [hunk 4]
```diff
@@ -54,10 +54,11 @@ This method supports some of the OData query parameters to help customize the re
 | Property     | Type        | Description |
```
Comment (FaithOmbongi) [inline api-reference/beta/api/organizationalbrandingproperties-create.md:54]:
```suggestion
In the request body, supply a JSON representation of [organizationalBrandingProperties](../resources/organizationalbrandingproperties.md) object. The following table lists the properties that are required when you create the default branding.

| Property     | Type        | Description |
```

**Review by FaithOmbongi — APPROVED** [review]

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
1. I recommend regenerating the examples to provide better context than 1/2 the properties having `null` values. If using the canary endpoint, replace that with the public endpoint in the docs.
2. List the properties in alphabetical order in both the table & JSON representation for scanability.
3. Find “ ”, replace with " ".
4. Don't forget the changelog.

**Pull request closed without merging**